src/GitVersionCore/ExecuteCore.cs

```diff
@@ -10,30 +8,50 @@
 
 namespace GitVersion
 {
-    public class ExecuteCore
+    public class ExecuteCore : IExecuteCore
```

Comment:
Is `ExecuteCore` a good name for a class? Wouldn't `CoreExecutor` or `GitVersionExecutor` be more descriptive and conventional?

Reply:
Changed to GitVersionExecutor